code/CwpInitialisationFilter.php

```diff
@@ -1,55 +1,84 @@
-	 *	requests should work out of the box. Stream-based requests need extra manual configuration.
-	 *	Refer to https://www.cwp.govt.nz/guides/core-technical-documentation/common-web-platform-core/en/how-tos/external_http_requests_with_proxy
-	 */
-	private static $egress_proxy_default_enabled = true;
-
-	/**
-	 * @var array Configure the list of domains to bypass proxy by setting the NO_PROXY environment variable.
-	 * 'services.cwp.govt.nz' needs to be present for Solr and Docvert internal CWP integration.
-	 * 'localhost' is necessary for accessing services on the same instance such as tika-server for text extraction.
-	 */
-	private static $egress_proxy_exclude_domains = array(
-		'services.cwp.govt.nz',
-		'localhost'
-	);
+class CwpInitialisationFilter implements RequestFilter
+{
 
-	public function preRequest(SS_HTTPRequest $request, Session $session, DataModel $model) {
+    /**
+     * @var boolean
+     *
+     * Enable egress proxy. This works on the principle of setting http(s)_proxy environment variables,
+     *  which will be automatically picked up by curl. This means RestfulService and raw curl
+     *  requests should work out of the box. Stream-based requests need extra manual configuration.
+     *  Refer to https://www.cwp.govt.nz/guides/core-technical-documentation/common-web-platform-core/en/how-tos/external_http_requests_with_proxy
+     *
+     * @config
+     * @var bool
+     */
+    private static $egress_proxy_default_enabled = true;
 
-		if (Config::inst()->get('CwpInitialisationFilter', 'egress_proxy_default_enabled')) {
+    /**
+     * @var array
+     *
+     * Configure the list of domains to bypass proxy by setting the NO_PROXY environment variable.
+     * 'services.cwp.govt.nz' needs to be present for Solr and Docvert internal CWP integration.
+     * 'localhost' is necessary for accessing services on the same instance such as tika-server for text extraction.
+     *
+     * @config
+     * @var string[]
+     */
+    private static $egress_proxy_exclude_domains = [
+        'services.cwp.govt.nz',
+        'localhost',
+    ];
 
-			if(defined('SS_OUTBOUND_PROXY') && defined('SS_OUTBOUND_PROXY_PORT')) {
-				putenv('http_proxy=' . SS_OUTBOUND_PROXY . ':' . SS_OUTBOUND_PROXY_PORT);
-				putenv('https_proxy=' . SS_OUTBOUND_PROXY . ':' . SS_OUTBOUND_PROXY_PORT);
-			}
+    /**
+     *
+     * @param  HTTPRequest $request
+     * @return boolean
+     */
+    public function preRequest(HTTPRequest $request)
+    {
+        if (Config::inst()->get(__CLASS__, 'egress_proxy_default_enabled')) {
+            if (Environment::getEnv('SS_OUTBOUND_PROXY') && Environment::getEnv('SS_OUTBOUND_PROXY_PORT')) {
+                $proxy = Environment::getEnv('SS_OUTBOUND_PROXY');
+                $proxyPort = Environment::getEnv('SS_OUTBOUND_PROXY_PORT');
 
-		}
+                Environment::setEnv('http_proxy', $proxy . ':' . $proxyPort);
+                Environment::setEnv('https_proxy', $proxy. ':' . $proxyPort);
```

Comment:
Does phpcs not take issue with the lack of space around the concatenation operator (left side, specifically)?

Reply:
Nope, it's not a rule in PSR-2